src/components/date/date.css

```diff
@@ -0,0 +1,43 @@
+@import '../../styles/spacing.css';
+@import '../../styles/colors-v2.css';
+
+.container {
+  position: relative;
+}
+
+.input {
+  --size-of-icons-container: 37px;
+
+  composes: input from '../input/input.css';
+  padding-right: calc(var(--size-of-icons-container) + var(--spacing-medium));
+}
+
+.input::-webkit-calendar-picker-indicator {
```

Comment:
Are there other vendor prefixes we need to cover as well?

Reply:
Nope! I tried it on our supported browser and it looked good! I think previous selector was copy-pasta from SO.